### DISCUSSION TITLE
AMP is calling getInitialProps or getServerSideProps several times

### DISCUSSION THREAD

**Discussion by yanv1991** [vercel/next.js#24246] [Help] [answered]
here is the repo of the reproducible issue: https://github.com/yanv1991/amp-issue

`
Home.getInitialProps = async (context) => {
  console.log('getInitialProps called');
  return { props: {} };
};
`

<img width="538" alt="Screen Shot 2021-04-19 at 8 28 49 PM" src="https://user-images.githubusercontent.com/3800032/115319453-dd29f400-a14d-11eb-9928-4a80c4b80499.png">

**Comment by gomflo** [comment]
Just ran your example and im not seeing that behavior

![image](https://user-images.githubusercontent.com/141670/115333317-1028b380-a15f-11eb-9fa2-4bbb19660612.png)

maybe you edit the documento and you confuse it with the fast refresh?

**Reply by yanv1991** [reply]
I used `yarn start`

**Comment by ijjk** [comment, marked as answer]
Hi, this sounds like a duplicate of https://github.com/vercel/next.js/issues/9212#issuecomment-547431272

**Reply by yanv1991** [reply]
@ijjk i disabled the extension and now is working as expected, thanks
